workflows/prognostic_c48_run/runtime/factories.py

```diff
@@ -150,7 +150,7 @@ def get_prescriber(
         mapper = {}
     time_lookup_function = _get_time_lookup_function(
         mapper,
-        list(config.variables),
+        list(config.variables) + list(config.tendency_variables or []),
```

Comment:
I'm thinking we should update the [prescriber tests](https://github.com/ai2cm/fv3net/blob/850965fe3b40128af1c3335c459cbda804968828/workflows/prognostic_c48_run/tests/test_prescriber.py#L132) to cover this (esp given that this slipped through).